### DISCUSSION TITLE
mitmproxy's certificate is broken

### DISCUSSION THREAD

**Discussion by Oreeeee** [mitmproxy/mitmproxy#5788] [Q&A]
#### Problem Description

Suddenly I can't use HTTPS with mitmproxy. Python Requests throws SSL error, curl throws `routines::invalid padding` error, everything worked prefectly yesterday.

#### Steps to reproduce the behavior:
1. Install mitmproxy's certificate
2.  Set up application/device to use mitmproxy
3. Try to connect through HTTPS with SLL verification enabled

#### System Information
Paste the output of "mitmproxy --version" here.

Mitmproxy: 9.0.1
Python:    3.10.8
OpenSSL:   OpenSSL 3.0.7 1 Nov 2022
Platform:  Linux-6.0.10-arch2-1-x86_64-with-glibc2.36

**Comment by ChillVibesMushroom** [comment]
How are you running sslstrip ping with mitm proxy are you running it in transparent mode

**Reply by Oreeeee** [reply]
I just run `mitmweb` (or `mitmproxy`) in the terminal to launch mitmproxy. In the logs I can see `Client TLS handshake failed. The client may not trust the proxy's certificate for google.com (OpenSSL Error([('SSL routines', '', 'tlsv1 alert decrypt error')]))`

**Comment by Oreeeee** [comment]
I only get this error with HTTP proxy, SOCKS5 works fine, but I need HTTP proxy because it's the only one supported by Android Studio

**Comment by Oreeeee** [comment]
The issue is still here, even on the latest version from the main branch.
